modules/installation-gcp-user-infra-rhcos.adoc

```diff
@@ -12,10 +12,8 @@ your {product-title} nodes.
 
 ifndef::openshift-origin[]
 . Obtain the {op-system} image from the
-link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red
-Hat customer portal or the
 link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/[{op-system} image mirror]
-page.
+page, under /pub/openshift-v4/dependencies/<op-system>/latest/.
```

Comment:
I don't think this makes sense. This text doesn't match the heading on the page, */pub/openshift-v4/dependencies/rhcos* or the link that you'd click. I think you should include /latest/ in the URL for the image mirror.

Reply:
@kalexand-rh Okay, yeah I see that. I think this needed to be changed in the other files too, so I went and made sure that /latest/ was added to the URL for image mirror

Reply:
Great! With that change, I think this makes more sense:
```suggestion
page.
```